FrontEnd/src/views/Schueler_Hauptmenue.js

```diff
@@ -151,6 +152,7 @@ export default function SchuelerHauptmenue(){
                     <Typography component="h1" variant="h4" color="inherit" noWrap className={classes.title}>
                         GradeMe
                     </Typography>
+                    <LogoutButton handleLogout={props.handleLogout} />
```

Comment:
Mach bitte einen Reducer für handleLogout und reich nicht eine Funktion als prop quer durch die app

Reply:
Würde es nicht mehr Sinn machen, alle Login/Logout Actions in einem Reducer zu behandeln? Dann kann man auch solche states wie loggedIn, die ja beide betreffen von einem Place aus ändern

Oder willst du nur für den Logout einen extra Reducer?